cmd/loop/loopout.go

```diff
@@ -145,9 +147,14 @@ func loopOut(ctx *cli.Context) error {
 		return err
 	}
 
-	var unchargeChannel uint64
-	if ctx.IsSet("channel") {
-		unchargeChannel = ctx.Uint64("channel")
+	chanStrings := strings.Split(ctx.String("channel"), ",")
+	var outgoingChanSet []uint64
+	for _, chanString := range chanStrings {
+		chanID, err := strconv.ParseUint(chanString, 10, 64)
```

Comment:
What if the the passed list/set contains duplicate items?

Reply:
This is now handled in `loopd`